### PR TITLE
Container Service Documentation

### PR DESCRIPTION
**Pull request by agngrant**
# EIDF Documentation Pull Request

## Description

Incorporation of Gitlab Documentation for the Container Service created by S Haynes into main EIDF documentation.

This is to prepare for the launch of the container service to a wider audience.

## Type of change

Please delete options that are not relevant.

- [x] New Documentation

## What has to be reviewed

Container Service - Overview
Container Service - Tutorial

## Checklist

- [x] Documentation follows the project style guidelines
- [x] Ensure Contact details contain Service Emails and Numbers
- [x] Self-review of documentation using mkdocs on local system
- [x] Spellcheck has been performed
- [x] Pre-commit has been run and passed

### CODE REVIEW

**Comment by agngrant** [comment]
Merge should be held until Launch

File: docs/services/containerservice/onboarding/new_user_walkthrough.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+All users need an account on the EIDF portal which provides users with a desktop VM to access the  GPU cluster.
+
+Full Documentation on signing up at [EIDF Documentation](https://epcced.github.io/eidf-docs/).
+
+a. Open Browser and goto the [EIDF Portal](https://portal.eidf.ac.uk)
+
+b. Click Login -> This will redirect to SAFE
+
+c. if you have a SAFE account -> use that account, if you do not have a SAFE account, register for a new SAFE account
```
Comment (kaleach) [inline docs/services/containerservice/onboarding/new_user_walkthrough.md:13]: Do we want to include a link into the SAFE docs? i.e. https://epcced.github.io/safe-docs/safe-for-users/#register

File: docs/services/containerservice/index.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+Multi-instance usage (MIG) GPUs allow multiple tasks or users to share the same GPU (similar to CPU threading).
+There are two types of MIG GPUs inside the EIDFCS the Nvidia A100 3G.20GB GPUs and the Nvidia A100 1G.5GB GPUs which equate to ~1/2 and ~1/7 of a full Nvidia A100 40 GB GPU.
+
+The current specification of the EIDFCS is:
+
+- 2368 CPU Cores
+- 12 TiB Memory
+- Local Disk Space (Node Image Cache and Local Workspace) - 28 TiB
+- Ceph Persistent Volumes (Long Term Data) - up to 100TiB
+- 32 Nvidia A100 80 GB GPUs
```
[the remark this answers is not in the document]
Comment (agngrant) [reply docs/services/containerservice/index.md:14]: Probably we should remove them - we would need to remove them from the Container Service Web Descriptor as well?

**Comment by agngrant** [comment]
Updated pytorch tutorial - AWatkins found an issue in a command, fix has been added.

File: docs/services/containerservice/index.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (agngrant) [reply docs/services/containerservice/index.md]: Updated this to reflect removal of the A100 80s. Along with reductions in core/memory.

**Comment by agngrant** [comment]
Updated pre-commit setup and mkdocs dev addr.

**Comment by agngrant** [comment]
Added quota to overview page.

**Pull request merged**